flink-contrib/flink-storm/src/test/java/org/apache/flink/storm/wrappers/StormTupleTest.java

```diff
@@ -613,29 +611,29 @@ public void testGetBinaryByFieldPojoGetter() throws Exception {
 		return new StormTuple(tuple, schema);
 	}
 
-	@Test(expected = UnsupportedOperationException.class)
+	@Test
 	public void testGetSourceGlobalStreamid() {
-		new StormTuple<Object>(null, null).getSourceGlobalStreamid();
+		Assert.assertNotNull(new StormTuple<Object>(null, null).getSourceGlobalStreamid());
```

Comment:
Can we improve on all this tests? Just a check for "not-null" seems a  little limited to me. We should rather check for defaults values and also check the full constructor case.